clearwater-infrastructure/usr/share/clearwater/bin/view-subscriber

```diff
@@ -69,8 +69,12 @@ def main():
 
                 ims_subscription = ET.tostring(reg_data.find('IMSSubscription'))
 
-                ccf_addresses = [(ccf.text, ccf.attrib['priority']) for ccf in reg_data.find('ChargingAddresses').findall('CCF')]
-                ecf_addresses = [(ecf.text, ecf.attrib['priority']) for ecf in reg_data.find('ChargingAddresses').findall('ECF')]
+                if reg_data.find('ChargingAddresses'):
```

Comment:
Is returning XML without a ChargingAddresses node valid? I didn't think it was, hence my code, and so I'm intrigued as to when we expect to hit it.